DEVX-2224: Heterogenous json structure recipe conversion (#640)
confluentinc/kafka-tutorials: _includes/tutorials/ksql-heterogeneous-json/ksql/markup/dev/problem-description.adoc

> Let's say you have a topic `source_data`, that contains JSON.  But each nested JSON object has a different structure.  Addtionally, in each object there is a mix of types within the values.

Reviewer: ```suggestion
Let's say you have a Kafka topic `source_data` that contains JSON-formatted data.  But each nested JSON object has a different structure.  Additionally, within each object the values have a mix of types.
```

Author: ack

> They each have one field in common that you want to pull out in a query and you don't care about the structure of the individual JSON objects

Reviewer: Given the example below, can we name the field that is common?

Author: changed the text some LMK WYT

[source, json]
----
{
  "JSONTypeOne": {
    "fieldA": "some data",
    "numberField": 1.001,
    "oneOnlyField": "more data", <1>
    "randomField": "random data"
  },

  "JSONTypeTwo": {
    "fieldA": "data",
    "fieldB": "b-data",
    "numberField": 98.6   <2>
  },

>   "JSONTypeThree": {

Reviewer: In my environment: I can't get this this third record to display in Safari unless I expand.  Scrolling does not help.

Can you please try in your view: do you see this 3rd record type?

Author: No, I have to expand as well.  It's like this with all KT's, maybe we need to explore the default of having it expand or allow scrolling.

Reviewer: As a workaround, what do you think about splitting each of the 3 JSON types into their own `[source, json]` code block?

Author: ack

    "fieldA": "data",
    "fieldB": "b-data",
    "numberField": 98.6,
    "fieldC": "data",
    "fieldD": "D-data"    <3>
  }
----
<1> The field you want from JSONTypeOne
<2> The field you want from JSONTypeTwo

> <3> The field you want form JSONTypeThree

Reviewer: ```suggestion
<3> The field you want from JSONTypeThree
```

Author: ack
